packages/vitepress-plugin-git-changelog/src/vite/helpers.ts

```diff
@@ -207,3 +207,7 @@ export function parseGitLogRefsAsTags(refs?: string): string[] {
 
   return tags.map(tag => tag.replace('tag: ', '').trim())
 }
+
+export function generateCommitPathsRegExp(includeDirs: string[], includeExtensions: `.${string}`[]): RegExp {
+  return new RegExp(`^${includeDirs.length > 0 ? `(${includeDirs.join('|')})${sep === win32.sep ? win32.sep : `\\${posix.sep}`}` : ''}.+${includeExtensions.length > 0 ? `(${includeExtensions.join('|')})` : '.md'}$`)
```

Comment:
I've removed the slash on this side because it's not compatible with the brackets generated by includeExtensions.

`.+\\.md$` => `.+.md$`